### DISCUSSION TITLE
Trial and activation issues

### DISCUSSION THREAD

**Discussion by kurtastrophe666** [rshea0/enjinn#58] [Help] [answered]
The 14 day trial doesn't work so I just bought and tried to activate and I am getting error on activation. Please help.

**Comment by shayded-exe** [comment]
So sorry for the issue. Please email me your license key at shayded@shayded.com and I'll send you an activated license file.

**Comment by shayded-exe** [comment, marked as answer]
This has been fixed in [v2.2.0](https://github.com/rshea0/tuneup-prime/releases/tag/v2.2.0)
